main.go

```diff
@@ -13,6 +12,7 @@ import (
 	"github.com/DataDog/datadog-api-client-go/v2/api/datadog"
 	"github.com/DataDog/datadog-api-client-go/v2/api/datadogV1"
 	"github.com/lmittmann/tint"
+	"github.com/pkg/errors"
```

Comment:
Is this wanted?

Reply:
mostly to wrap errors, such as: https://github.com/persona-id/datadog-query-linter/pull/8/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R157